crates/update/src/cli/install.rs

```diff
@@ -125,6 +125,7 @@ pub(super) async fn download_and_install(
         ArtifactType::Zip => {
             let zip = archive.to_bytes();
             let zip = io::Cursor::new(&*zip);
+            version_dir.create()?;
```

Comment:
I think it's better to not depend on a library to do this for us in either case. Can we move it outside of the `match`? I will test on Ubuntu.

Reply:
Yeah I'm fine with that